src/styles.scss

```diff
@@ -266,6 +266,12 @@ button,
   height: 100%;
 }
 
+.App-menu__left {
```

Comment:
Could this be done in a way that makes the `Island` take the role of the parent element so there aren’t duplicated shadows?

Reply:
Changed this class to be assigned to Island.